DFReader.py

```diff
@@ -758,16 +758,20 @@ def init_arrays(self, progress_callback=None):
         while ofs+3 < self.data_len:
             hdr = self.data_map[ofs:ofs+3]
             if hdr[0] != HEAD1 or hdr[1] != HEAD2:
-                print("bad header 0x%02x 0x%02x" % (u_ord(hdr[0]), u_ord(hdr[1])), file=sys.stderr)
+                # avoid end of file garbage, block based logs can have up to 256 bytes of zeros
```

Comment:
The 256 and the 528 are at odds with one-another here.  There might be a good reason for, but if it's making me curious it probably needs a comment at least :-)

Reply:
Comment expanded